### PR TITLE
p2p network devops refactor

### PR DESCRIPTION
**Pull request by steevi**
develop

### CODE REVIEW

File: scripts/acn/helm-chart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+keywords:
+- fetch
+- agents
+home: https://fetch.ai
+sources:
+- https://github.com/fetchai
+maintainers:
+- name: steevi
+  email: stefanos.malliaros@fetch.ai
+icon: https://avatars3.githubusercontent.com/u/40889903
```
Comment (DavidMinarsch) [inline scripts/acn/helm-chart/Chart.yaml:15]: Intentionally hardcoded?

**Review by DavidMinarsch — APPROVED** [review]
LGTM, one comment

File: scripts/acn/helm-chart/Chart.yaml (hunk 1 above)
Comment (steevi) [reply scripts/acn/helm-chart/Chart.yaml:15]: No. I'll delete. That i copy-pasted from our infra repos, which we would like to know who contributed to which helm charts

**Review by DavidMinarsch — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**